### DISCUSSION TITLE
How do I increase the number of rows in each page in a table ?

### DISCUSSION THREAD

**Discussion by dittorahmat** [nocodb/nocodb#942] [Q&A] [answered]
How do I increase the number of rows in each page in a table (not in rest API) ? Currently there are 25 rows per page in a table. Is there anyway to increase this limit ?

I tried looking at the docs and can only find pagination & offset on rest API

**Comment by wingkwong** [comment, marked as answer]
The default limit is calculated as ``max(min(DB_QUERY_LIMIT_DEFAULT , DB_QUERY_LIMIT_MAX), DB_QUERY_LIMIT_MIN)``. By default the initial values for ``DB_QUERY_LIMIT_DEFAULT``, ``DB_QUERY_LIMIT_MAX``, and ``DB_QUERY_LIMIT_MIN`` are ``25``, ``100``, and ``1`` respectively. 

Therefore, you need to increase ``DB_QUERY_LIMIT_DEFAULT``. If your target ``DB_QUERY_LIMIT_DEFAULT`` exceeds the default value ``100``, then you need to increase ``DB_QUERY_LIMIT_MAX`` as well. 

For example, setting ``DB_QUERY_LIMIT_DEFAULT`` to ``50`` would have the following result:

![image](https://user-images.githubusercontent.com/35857179/148919209-a64c111e-ecdc-46a5-b7ca-e4e4ac2291f6.png)

**Comment by pedrorito** [comment]
In version 0.96.4 `DB_QUERY_LIMIT_DEFAULT` seems to not be working anymore. I have set mine to 100, and from this version I only have 25 rows per page.

**Reply by o1lab** [reply]
issue created here : https://github.com/nocodb/nocodb/issues/3617
